src/AuthProvider.tsx

```diff
@@ -121,6 +150,19 @@ const unsupportedEnvironment = (fnName: string) => () => {
 const UserManagerImpl =
     typeof window === "undefined" ? null : UserManager;
 
+const defaultMatchSignoutCallback =
+(props: { userManager?: UserManager; post_logout_redirect_uri?: string; popup_post_logout_redirect_uri?: string }) => (): boolean => {
+    const {
+        post_logout_redirect_uri,
+        popup_post_logout_redirect_uri,
+    } = props.userManager ? props.userManager.settings : props;
+    return (
+        window &&
+        (window.location.pathname === post_logout_redirect_uri ||
+         window.location.pathname === popup_post_logout_redirect_uri)
```

Comment:
I think usually the uri would be not just the path, but the full url including the host name, right?

Reply:
I think I would have a look into some oauth2 spec or something. I would expect that just a pathname as suffix is generally not allowed and I would just compare the full url via `===`. So I would expect something like `window.location.href === post_logout_redirect_uri`.

Maybe validate that the `post_logout_redirect_uri` also contains the protocol and host name. Especially as this should be a query parameter on the oidc logout call and for this it is probably anyway required that the oidc server knows the exact url. Otherwise no redirect from that server would be possible.

Reply:
if `post_logout_redirect_uri` has no protocol or host, the IDP can not callback. The `window.location.href` contains the protocol and host.